thunder/core/interpreter.py

```diff
@@ -2101,8 +2101,9 @@ def __new__(cls, iterable=()):
         return wrap_const(cls.value())
 
     def __init__(self, iterable=()):
-        SequenceWrapperMethods.__init__(self, iterable)
-        return wrap_const(None)
+        # We need to propagate the return value because it could be JIT_SIGNALS
+        res = SequenceWrapperMethods.__init__(self, iterable)
+        return res
```

Comment:
```suggestion
        return SequenceWrapperMethods.__init__(self, iterable)
```

Reply:
Yeah, we could, but I exect to add debug prints soonish.